app/vmbackupmanager/README.md

```diff
@@ -377,7 +377,7 @@ command-line flags:
   -dst string
      The root folder of Victoria Metrics backups. Example: gs://bucket/path/to/backup/dir, s3://bucket/path/to/backup/dir or fs:///path/to/local/backup/dir
   -enableTCP6
-     Whether to enable IPv6 for listening and dialing. By default only IPv4 TCP and UDP is used
+     Whether to enable IPv6 for listening and dialing. By default, only IPv4 TCP and UDP is used
```

Comment:
NIT: if you want to make it works with flags you need to change the code as well, otherwise next flag generation will replace this again. However you can skip it as well